src/bundle/wysiwyg-preset.ts

```diff
@@ -63,9 +64,26 @@ export const BundlePreset: ExtensionAuto<BundlePresetOptions> = (builder, opts)
         baseSchema: {
             paragraphKey: f.toPM(A.Text),
             paragraphPlaceholder: (node: Node, parent?: Node | null) => {
-                const isDocEmpty =
-                    !node.text && parent?.type.name === BaseNode.Doc && parent.childCount === 1;
-                return isDocEmpty ? i18nPlaceholder('doc_empty') : null;
+                let isPlaceholderShown = false;
+
+                if (opts.placeholderOptions?.behavior === 'empty-row') {
+                    isPlaceholderShown = !node.text && parent?.type.name === 'doc';
```

Comment:
`parent?.type.name === 'doc'` – this means that placeholder will be displayed only if empty paragraph is direct child of doc (is located top level in document tree).

Perhaps when `behavior:'empty-row'` placeholder should be diplayed in any empty paragraph? For example, inside notes, cuts, qoutes and others?

Reply:
The idea is that we should only consider an empty string to be present if it is actually part of the document. We do not want to display placeholders inside blocks